README.md

```diff
@@ -1,15 +1,18 @@
+![Alt text](https://assets-global.website-files.com/5e21dc6f4c5acf29c35bb32c/5e21e66410e34945f7f25add_Keboola_logo.svg)
+
```

Comment:
We don't to this usually (even if I like logo much). Rather at there a link to passing tests ;)